includes/flattened_entity_version.js

```diff
@@ -81,28 +82,71 @@ module.exports = (params) => {
         coalesceSql = `DATA_struct.${key.keyName}`;
       }
       else {
-        coalesceSql = `COALESCE(DATA_struct.${key.keyName}, DATA_struct.${key.pastKeyNames.join(', DATA_struct.')})`;
+        coalesceSql = `
+            COALESCE(DATA_struct.${
+                key.keyName
+            }, DATA_struct.${
+                key.pastKeyNames.join(', DATA_struct.')
+            })`;
       }
       let fieldSql;
       if (key.dataType == 'boolean') {
-        fieldSql = `SAFE_CAST(${coalesceSql} AS BOOL)`;
+        fieldSql = `
+            SAFE_CAST(${
+                    coalesceSql
+                }
+                AS BOOL)`;
       } else if (key.dataType == 'timestamp') {
-        fieldSql = `${data_functions.stringToTimestamp(coalesceSql)}`;
+        fieldSql = `
+            ${
+                data_functions.stringToTimestamp(coalesceSql)
+            }
+            `;
       } else if (key.dataType == 'date') {
-        fieldSql = `${data_functions.stringToDate(coalesceSql)}`;
+        fieldSql = `
+            ${
+                data_functions.stringToDate(coalesceSql)
+            }
+            `;
       } else if (key.dataType == 'integer') {
-        fieldSql = `SAFE_CAST(${coalesceSql} AS INT64)`;
+        fieldSql = `
+            SAFE_CAST(${
+                    coalesceSql
+                }
+                AS INT64)`;
       } else if (key.dataType == 'integer_array') {
-        fieldSql = `${data_functions.stringToIntegerArray(coalesceSql)}`;
+        fieldSql = `
+            ${
+                data_functions.stringToIntegerArray(coalesceSql)
+            }
+            `;
       } else if (key.dataType == 'float') {
-        fieldSql = `SAFE_CAST(${coalesceSql} AS FLOAT64)`;
+        fieldSql = `
+            SAFE_CAST(${
+                    coalesceSql
+                }
+                AS FLOAT64)`;
       } else if (key.dataType == 'json') {
-        fieldSql = `SAFE.PARSE_JSON(${coalesceSql})`;
+        fieldSql = `
+            SAFE.PARSE_JSON(${
+                coalesceSql
+            })`;
       } else if (key.dataType == 'string' || key.dataType == undefined) {
-        fieldSql = `${coalesceSql}`;
+        fieldSql = `
+            ${
+                coalesceSql
+            }
+            `;
```

Comment:
The dev branch looked fine, but wanted to check that the format change on multiple lines rather than 1 won't generate issues further down stream.

I've had it before where I've hit format too earlier and a missing '}' has meant that the the all the references with `${example}` change. This might be okay though, but thought best to check

Reply:
I'll change it back anyway as I think it's more readable not spread over multiple lines